src/pretix/static/pretixcontrol/scss/_forms.scss

```diff
@@ -2,6 +2,14 @@
   color: $text-muted;
 }
 
+.content-placeholder {
+  padding-left: 0;
+  padding-right: 0;
+  display: inline;
+  background: none;
+  border: 0;
```

Comment:
Should we make them look more clickable? Currently it's hard to disover it. Either make them look more like buttons, or a "click to insert" (as part of the `title`?)

Reply:
I had them styled as buttons initially, but that was a wall of buttons with lots of grey lines (btn-default). So chose not to style as buttons. I tried `btn-link`, which was okay (except alignment issues with the text abseline of „Available placholders:“) What about a new style? Not sure about introducing a new style, but maybe this:

![Bildschirmfoto 2023-09-12 um 20 18 03](https://github.com/pretix/pretix/assets/276509/8087745c-ac0c-4147-829c-fdd88c558f30)

Reply:
I think this looks nice. It's not too distracting but still makes clear that these are special and you can do something with them.

Reply:
I added that light button style. I also removed the comma in between each button element to A) reduce visual clutter and B) in certain conditions, the line break was happening before the comma and having a comma at the beginning of a line looks a bit unusual.